src/axom/slic/docs/sphinx/sections/appendix.rst

```diff
@@ -67,332 +67,34 @@ functions are called.
+| | ``SLIC_INFO``            | | No       | |                                                                        |
+| | ``SLIC_INFO_IF``         | |          | |                                                                        |
+| | ``SLIC_INFO_ROOT``       | |          | |                                                                        |
+| | ``SLIC_INFO_ROOT_IF``    | |          | |                                                                        |
+|                            |            |                                                                          |
+| | ``SLIC_ERROR``           | | Yes      | | Not collective after ``slic::disableAbortOnError()`` is called         |
+| | ``SLIC_ERROR_IF``        | |          | |                                                                        |
+| | ``SLIC_ERROR_ROOT``      | |          | |                                                                        |
+| | ``SLIC_ERROR_ROOT_IF``   | |          | |                                                                        |
+|                            |            |                                                                          |
+| | ``SLIC_WARNING``         | | Yes      | | Collective after ``slic::enableAbortOnWarning()`` is called            |
+| | ``SLIC_WARNING_IF``      | |          | |                                                                        |
+| | ``SLIC_WARNING_ROOT``    | |          | |                                                                        |
+| | ``SLIC_WARNING_ROOT_IF`` | |          | |                                                                        |
+|                            |            |                                                                          |
+| | ``SLIC_DEBUG``           | | No       | |                                                                        |
+| | ``SLIC_DEBUG_IF``        | |          | |                                                                        |
+| | ``SLIC_DEBUG_ROOT``      | |          | |                                                                        |
+| | ``SLIC_DEBUG_ROOT_IF``   | |          | |                                                                        |
+|                            |            |                                                                          |
+| | ``SLIC_ASSERT``          | | Yes      | | Not collective after ``slic::disableAbortOnError()`` is called         |
+| | ``SLIC_ASSERT_MSG``      | |          | |                                                                        |
+|                            |            |                                                                          |
+| | ``SLIC_CHECK``           | | Yes      | | Collective after ``slic::debug::checksAreErrors`` is set to true       |
+| | ``SLIC_CHECK_MSG``       | |          | |                                                                        |
```

Comment:
Thanks for simplifying this!

Could we clarify it a bit more?
E.g. change the `Collective` column to `Collective (by default)`

And update the notes to indicate which calls make them collective and which ones remove it.
E.g. for `SLIC_ERROR`, I think it would be:
```
Collective by default, and after calling `slic::enableAbortOnError()`.
No longer collective after calling `slic::disableAbortOnError()`
```